activity.py

```diff
@@ -42,28 +42,27 @@ def build_toolbar(self):
         toolbar_box.toolbar.insert(activity_button, -1)
         activity_button.show()
 
-        separator = Gtk.SeparatorToolItem()
-        separator.props.draw = False
-        separator.set_expand(False)
-        toolbar_box.toolbar.insert(separator, -1)
-        separator.show()
+        self.add_toolbar_separator(toolbar_box)
 
         button = ToolButton('speaker-muted-100')
         button.set_tooltip(_('Sound'))
         button.connect('clicked', self.sound_control)
         toolbar_box.toolbar.insert(button, -1)
 
-        separator = Gtk.SeparatorToolItem()
-        separator.props.draw = False
-        separator.set_expand(True)
-        toolbar_box.toolbar.insert(separator, -1)
-        separator.show()
+        self.add_toolbar_separator(toolbar_box, expand=True)
 
         stop_button = StopButton(self)
         toolbar_box.toolbar.insert(stop_button, -1)
         stop_button.show()
         stop_button.connect('clicked', self._stop_cb)
 
+    def add_toolbar_separator(self, toolbar_box, expand=False):
+        separator = Gtk.SeparatorToolItem()
+        separator.props.draw = False
+        separator.set_expand(expand)
+        toolbar_box.toolbar.insert(separator, -1)
+        separator.show()
+
```

Comment:
A better way would be to make toolbar_box and instance variable and not pass it as an arg as I don't see any particular reason why that happens.